data/mods/ssb/moves.js

```diff
@@ -3329,6 +3329,21 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Fairy",
 	},
+	// Used for PiratePrincess's ability
+	acidrain: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "For 5 turns, the weather becomes Acid Rain.  Pokemon that are not Poison-type take damage every turn.  Special Defense of Poison-type pokemon is multiplied by 1.5.  Poison moves ignore Steel immunity and Poison-type Pokemon can be poisoned.",
+		shortDesc: "5 turns: +Poison SpD, corrosive damage.",
+		id: "acidrain",
+		name: "Acid Rain",
+		isNonstandard: "Custom",
+		pp: 5,
+		priority: 0,
```

Comment:
flags: {},
weather: 'acidrain',
onTryMove() {
	this.attrLastMove('[still]');
},
onPrepareHit(target, source) {
	this.add('-anim', source, 'Rain Dance', source);
},
secondary: null,